docs/other-helpers.md

```diff
@@ -28,11 +38,51 @@ llm.PullModel(url, model string) (llm.PullResult, int, error)
 `PullModel` sends a POST request to the specified URL to pull a model with the given name.
 
 **Parameters**:
-  - url: The URL to send the request to.
-  - model: The name of the model to pull.
+
+  - `url`: The URL to send the request to.
+  - `model`: The name of the model to pull.
 
 **Returns**:
-  - PullResult: The result of the pull operation.
-  - int: The HTTP status code of the response.
-  - error: An error if the request fails.
-<!-- split -->
\ No newline at end of file
+
+  - `PullResult`: The result of the pull operation.
+  - `int`: The HTTP status code of the response.
+  - `error`: An error if the request fails.
+
+> If you use a protected Ollama endpoint, use this function:
+> 
+> ```golang
+> llm.PullModelWithToken(url, model , tokenHeaderName, tokenHeaderValue string) (llm.PullResult, int, error)
+> ```
+
+<!-- split -->
+
+## Get the list of the installed models
+
+```golang
+llm.GetModelsList(url string) (llm.ModelList, int, error)
+```
+
+`GetModelsList` sends a GET request to the specified URL to fetch the list of the installed models.
+
+**Parameters**:
+
+  - `url`: The URL to send the request to.
+
+**Returns**:
+
+  - `ModelList`: The result of the reques, use the `models` property to get the list.
+  - `int`: The HTTP status code of the response.
+  - `error`: An error if the request fails.
+
+> If you use a protected Ollama endpoint, use this function:
```

Comment:
curious if you set a token to empty and it isn't a protected endpoint, would it still work?

Reply:
it should work